knimin/templates/pm_create_plate.html

```diff
@@ -0,0 +1,124 @@
+{% extends logged_in_index.html %}
+{% block head %}
+
+
+<script type="text/javascript">
+
+  function add_study_options(selDom) {
+    var studies = {% raw [ [int(s[0]), s[1], s[2], s[3]] for s in studies] %};
```

Comment:
possible to expand the list so that variable names are associated instead of indices?